[2023-08-17T04:10:27.573+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2023-08-17T04:10:26.463987+00:00 [queued]>
[2023-08-17T04:10:27.580+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2023-08-17T04:10:26.463987+00:00 [queued]>
[2023-08-17T04:10:27.581+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-17T04:10:27.589+0000] {taskinstance.py:1327} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2023-08-17 04:10:26.463987+00:00
[2023-08-17T04:10:27.592+0000] {standard_task_runner.py:57} INFO - Started process 1809 to run task
[2023-08-17T04:10:27.597+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'spark_***_dag', 'spark_submit_task', 'manual__2023-08-17T04:10:26.463987+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/spark_jobs_dag.py', '--cfg-path', '/tmp/tmp35smc21c']
[2023-08-17T04:10:27.598+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask spark_submit_task
[2023-08-17T04:10:27.651+0000] {task_command.py:410} INFO - Running <TaskInstance: spark_airflow_dag.spark_submit_task manual__2023-08-17T04:10:26.463987+00:00 [running]> on host 0d68e811d1ff
[2023-08-17T04:10:27.721+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_***_dag' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-17T04:10:26.463987+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-17T04:10:26.463987+00:00'
[2023-08-17T04:10:27.733+0000] {base.py:73} INFO - Using connection ID 'spark_local' for task execution.
[2023-08-17T04:10:27.734+0000] {spark_submit.py:340} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --name arrow-spark /dags/spark_processing.py
[2023-08-17T04:10:27.821+0000] {spark_submit.py:491} INFO - /home/***/.local/lib/python3.10/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2023-08-17T04:10:29.097+0000] {spark_submit.py:491} INFO - python3: can't open file '/dags/spark_processing.py': [Errno 2] No such file or directory
[2023-08-17T04:10:29.107+0000] {spark_submit.py:491} INFO - 23/08/17 04:10:29 INFO ShutdownHookManager: Shutdown hook called
[2023-08-17T04:10:29.109+0000] {spark_submit.py:491} INFO - 23/08/17 04:10:29 INFO ShutdownHookManager: Deleting directory /tmp/spark-d77bf98e-f693-4adc-97bc-954fffc83b8b
[2023-08-17T04:10:29.182+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 156, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 422, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --name arrow-spark /dags/spark_processing.py. Error code is: 2.
[2023-08-17T04:10:29.186+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=spark_***_dag, task_id=spark_submit_task, execution_date=20230817T041026, start_date=20230817T041027, end_date=20230817T041029
[2023-08-17T04:10:29.195+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 12 for task spark_submit_task (Cannot execute: spark-submit --master spark://spark:7077 --name arrow-spark /dags/spark_processing.py. Error code is: 2.; 1809)
[2023-08-17T04:10:29.215+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-08-17T04:10:29.233+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
